ChaosMod/Effects/db/Vehs/VehsDamageController.cpp

```diff
@@ -0,0 +1,86 @@
+#include "stdafx.h"
+
+// clang-format off
+static void OnStopInvincible()
+{
+	for (auto veh : GetAllVehs())
+	{
+		SET_ENTITY_INVINCIBLE(veh, false);
+	}
+}
+
+static void OnTickInvincible()
+{
+	for (auto veh : GetAllVehs())
+	{
+		SET_ENTITY_INVINCIBLE(veh, true);
+	}
+}
+
+REGISTER_EFFECT(nullptr, OnStopInvincible, OnTickInvincible, EffectInfo // By pongo1231
+	{
+		.Name = "All Vehicles Are Invulnerable",
+		.Id = "vehs_invincible",
+		.IsTimed = true,
+        .IncompatibleWith = { "vehs_ohko", "veh_speed_goal", "vehs_damage_x2", "vehs_damage_x5", "vehs_damage_x05" }
+	}
+);
```

Comment:
```suggestion
// clang-format off
REGISTER_EFFECT(nullptr, OnStopInvincible, OnTickInvincible, EffectInfo // By pongo1231
	{
		.Name = "All Vehicles Are Invulnerable",
		.Id = "vehs_invincible",
		.IsTimed = true,
        .IncompatibleWith = { "vehs_ohko", "veh_speed_goal", "vehs_damage_x2", "vehs_damage_x5", "vehs_damage_x05" }
	}
);
// clang-format on
```
Same for the ones below